ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobStoreRecoveryTest.java

```diff
@@ -66,16 +66,19 @@ public String getLongForm() {
     return getID();
   }
 
-  @Override
   public short getAccountId() {
```

Comment:
you can remove these two functions from here - they were added as part of the original change

Reply:
fixed.